train.py

```diff
@@ -95,9 +95,6 @@ def train(args):
                 start = time.time()
                 x, y = data_loader.next_batch()
                 feed = {model.input_data: x, model.targets: y}
-                for i, (c, h) in enumerate(model.initial_state):
```

Comment:
I agree with you, filed an issue https://github.com/tensorflow/models/issues/774